oneBarangay/static/assets/js/volt.js

```diff
@@ -260,96 +260,96 @@ d.addEventListener("DOMContentLoaded", function (event) {
       donutSolid: true,
       fullWidth: false,
       showLabel: false,
-      plugins: [Chartist.plugins.tooltip()],
-    });
+      plugins: [Chartist.plugins.tooltip()]
+    })
   }
 
-  if (d.getElementById("loadOnClick")) {
-    d.getElementById("loadOnClick").addEventListener("click", function () {
-      const button = this;
-      const loadContent = d.getElementById("extraContent");
-      const allLoaded = d.getElementById("allLoadedText");
+  if (d.getElementById('loadOnClick')) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
    if (d.getElementById("loadOnClick")) {
```